microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
 You can dock DevTools in the right, left, or bottom of the browser, or undock it into a separate window.  See [Change DevTools placement (Undock, Dock to bottom, Dock to left)](customize/placement.md).
 
 
+<!-- ====================================================================== -->
+## Zooming DevTools or the rendered page
+
+The DevTools UI is implemented using HTML and CSS, and you can zoom it in and out like any other web page.  The zoom level for the DevTools portion of the browser is independent from the zoom level for the rendered webpage portion of the browser.
```

Comment:
I'd like to suggest a slight rephrasing to make the relationship between DevTools using HTML/CSS and the fact that you can zoom it more obvious:

"The DevTools UI is implemented using HTML and CSS, like web pages, which means you can zoom it in and out like any other web page."